ouroboros-network/ouroboros-network.cabal

```diff
@@ -184,6 +184,57 @@ library
   if flag(txsubmission-delay)
     cpp-options: -DTXSUBMISSION_DELAY
 
+library ouroboros-orphan-instances
+  import: ghc-options
+  visibility: public
+  hs-source-dirs: ouroboros-orphan-instances
+  exposed-modules:
+    Ouroboros.Cardano.Network.OrphanInstances
+    Ouroboros.Network.OrphanInstances
+
+  other-modules:
+  reexported-modules:
+  default-language: Haskell2010
+  default-extensions: ImportQualifiedPost
+  other-extensions:
+    BangPatterns
+    DataKinds
+    EmptyCase
+    ExistentialQuantification
+    FlexibleContexts
+    FlexibleInstances
+    FunctionalDependencies
+    GADTSyntax
+    GADTs
+    GeneralizedNewtypeDeriving
+    MultiParamTypeClasses
+    NamedFieldPuns
+    OverloadedStrings
+    PolyKinds
+    RankNTypes
+    RecordWildCards
+    ScopedTypeVariables
+    TemplateHaskell
+    TupleSections
+    TypeApplications
+    TypeFamilies
+    TypeInType
+
+  build-depends:
+    aeson,
+    base >=4.14 && <4.22,
+    containers,
+    iproute,
+    network,
+    network-mux,
+    ouroboros-network:{ouroboros-network, cardano-diffusion},
+    ouroboros-network-api ^>=0.13,
+    ouroboros-network-framework ^>=0.17,
```

Comment:
I think `ouroboros-network-framework` includes a constraint on `ouroboros-network-api`; if it indeed does, we don't need to constrain `o-n-api` again, which makes it simpler when we update versions.

Reply:
it does not